cime_config/tests.py

```diff
@@ -606,7 +606,7 @@
     "e3sm_scream_v1_hires" : {
         "time"  : "01:00:00",
         "tests" : (
-            "SMS_Ln90.ne30pg2_ne30pg2.F2010-SCREAMv1.scream-perf_test"
+            "SMS_Ln300.ne30pg2_ne30pg2.F2010-SCREAMv1.scream-perf_test"
```

Comment:
Should we also add higher res (with higher node count) like done in the `e3sm_bench_hires_g` testsuite?

Reply:
I can add whatever the team wants. This is just a starting point.